### ISSUE TITLE
StEaLiNg?!?!?!!?

### ISSUE THREAD

**Issue by TrickJoker312** [Realhack101/Realhack101-s-Unblocked-Games#1]
Do any of these links happen to come from my repository?

**Comment by Realhack101** [comment]
yes some these are mostly for my friends in my school and  these all work for my district, links are not stealing if these are your games idk we can talk abt that.

**Comment by TrickJoker312** [comment]
Ok, I'd appreciate it you could let me know next time you use the links.

**Comment by Realhack101** [comment]
okay thank you
